idd/Energy+.idd.in

```diff
@@ -7604,6 +7604,58 @@ MaterialProperty:VariableThermalConductivity,
        \units W/m-K
        \type real
 
+MaterialProperty:VariableAbsorptance,
+  A1 , \field Name
+       \required-field
+       \type alpha
+       \note The name of the dynamic coating material with variable thermal absorptance.
+  A2 , \field Reference Material Name
+       \required-field
+       \type object-list
+       \object-list MaterialName
+       \note Regular Material Name to which the additional properties will be added.
+       \note this the material name for the basic material properties.
+  A3 , \field Control Signal
+       \required-field
+       \type choice
+       \key SurfaceTemperature
+       \key SurfaceReceivedSolarRadiation
+       \key SpaceHeatingCoolingMode
+       \key Scheduled
+       \default SurfaceTemperature
+  A4 , \field Trigger Thermal Absorptance Function Name
```

Comment:
What is the word "Trigger" referring to here?

Reply:
It refers to the control signal, I'll change it to "Control Signal-Thermal Absorptance Function Name"

Reply:
You can just delete it also. It’s not necessary.

Reply:
oh that works too. Maybe I'll just delete it.